web/client/components/map/openlayers/Layer.jsx

```diff
@@ -49,7 +51,14 @@ export default class OpenlayersLayer extends React.Component {
         this.valid = true;
         this.tilestoload = 0;
         this.imagestoload = 0;
-        this.createLayer(this.props.type, this.props.options, this.props.position, this.props.securityToken);
+        this.createLayer(
```

Comment:
If possible I would have added a generic ENV parameter here, and handled the env params for localization at an higher level. What happens if we want to use ENV for other purposes?

Reply:
@mbarto so if I understood you correctly you want createLayer functions tolook like this
```
this.createLayer(
            this.props.type,
            this.props.options,
            this.props.position,
            this.props.securityToken,
            this.props.env
        );
```
so the last to params will be combined and formed on the upper level. Am I right?

Reply:
Yes, exactly, this would leave the Layer open to accept env for other goals